src/Service/JiraApiService.php

```diff
@@ -47,7 +50,7 @@ class JiraApiService implements ApiServiceInterface, ProjectTrackerInterface
     private const API_PATH_ACCOUNT_IDS_BY_PROJECT = '/rest/tempo-accounts/1/link/project/';
 
     public function __construct(
-        protected readonly HttpClientInterface $jiraProjectTrackerApi,
+        protected readonly HttpClientInterface $projectTrackerApi,
```

Comment:
Change the name back to $jiraProjectTrackerApi

This name cannot be changed without changing the name of the scoped client:
https://github.com/itk-dev/economics/blob/develop/config/packages/framework.yaml#L22